.github/workflows/ci.yaml

```diff
@@ -2,6 +2,9 @@
 name: Build VS Code Extension
 
 on:
+  release:
+    types:
+      - published
```

Comment:
I set this up to use a _release_ but alternatively we could [use a tag in the way the Shiny team does](https://github.com/posit-dev/shiny-vscode/blob/main/.github/workflows/publish.yml).